pkgs/top-level/all-packages.nix

```diff
@@ -14981,6 +14981,8 @@ with pkgs;
 
   zsh-vi-mode = callPackage ../shells/zsh/zsh-vi-mode { };
 
+  zsh-wd = callPackage ../shells/zsh/zsh-wd { };
+
```

Comment:
This fails the `pkgs/by-name` check but is grouping this plugin with other zsh plugins. Let me know if it should be moved though!

Reply:
Please put the package under `pkgs/by-name`
See `pkgs/by-name/README.md`

Reply:
Thanks for checking this - updated! Wasn't sure if that still applied with the other plugins being here 😅